scheduling-manager/src/cucumber/resources/features/scheduling/replace_after_terminate.feature

```diff
@@ -37,3 +37,29 @@ Feature: Deploying a Daemon environment to a cluster with Replace After Terminat
       | containerInstanceArn | taskDefinitionArn | group             |
       | i-1                  | v1                | DaemonEnvironment |
     And it should not take any further actions
+
+  Scenario: Scheduling on a cluster with tasks that don't match the environment version
+    Given the cluster has the following instances and tasks:
+      | instance | tasks          |
+      | i-1      | t-1:v1:RUNNING |
+      | i-2      | t-2:v2:RUNNING |
+    When the scheduler runs
+    Then it should stop the following tasks:
+      | clusterName | task           | reason                |
+      | TestCluster | v2             | Stopped by deployment to DaemonEnvironment@v1 |
```

Comment:
This test case might be easier if we make a new feature file, where the current version of the environment is `v2`, so that versions are changing forward in time. Perhaps consider splitting these into:

```
features/replace_after_terminate/initial_deployment.feature
features/replace_after_terminate/update_deployment.feature
```

Reply:
I tried separating it out and figure that there are only two test scenarios in the new file. If we consider adding more to the update_deployment one that'll be a good idea